tests/Rector/v11/v3/ReplaceStdAuthCodeWithHmacRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v11\v3\ReplaceStdAuthCodeWithHmacRector\Fixture;
+
+\TYPO3\CMS\Core\Utility\GeneralUtility::stdAuthCode(5);
+
+?>
+-----
```

Comment:
If it is not touched remove the code under the dashes.

Reply:
changed